online_store/gt_sport_v2.js

```diff
@@ -157,11 +138,12 @@ function addProductsToCart(product) {
 
 function displayContentInCart() {
   for (let i = 0; i < shoppingCart.length; i++) {
```

Comment:
Before for loop you need to clean the innerHTML of shoppingCartEl. Otherwise you duplicate content inside.

Reply:
Yep, thanks, didn't notice!
Fixed now!